thirdparty/dependencies.cmake

```diff
@@ -304,7 +305,10 @@ endif()
 
 if(ENABLE_SAMPLES OR ENABLE_TESTS OR ENABLE_INTEL_NPU_INTERNAL)
     add_subdirectory(thirdparty/gflags EXCLUDE_FROM_ALL)
-    ov_developer_package_export_targets(TARGET gflags)
+    ov_developer_package_export_targets(
+        TARGET gflags
+        INSTALL_INCLUDE_DIRECTORIES "${CMAKE_BINARY_DIR}/thirdparty/gflags/gflags/include/gflags"
+        INSTALL_DESTIONATION "developer_package/include/gflags")
```

Comment:
`INSTALL_DESTIONATION` is a typo from the `ov_developer_package_export_targets` function.
I propose addressing this typo (and any affected files) in another PR.